ParseNOTAM.py

```diff
@@ -10,15 +10,14 @@ def ParseNOTAM(json_data = None):
 
         # Reading JSON data from the file
         with open(file_path, 'r') as json_file:
-            json_data = json.load(json_file)
+            json_data_test = json.load(json_file)
 
-
-    # create a class for each NOTAM
     NOTAMs = []
-    for item in json_data['items']:
-        NOTAMs.append(Models.Notam(item['properties']['coreNOTAMData']['notam']))
-        
+    for notam in json_data:

```

Comment:
seems odd to loop over every field in `json_data` and then for each one loop over over child in its `items` field ... perhaps a comment here about the expected structure of the JSON data (or even a short example) is warranted?